catalog/ui/search-ui/standard/src/main/webapp/js/view/filter/FilterItem.view.js

```diff
@@ -63,7 +64,13 @@ define([
                     wreqr.vent.trigger('search:drawend');
                 }
             },
-
+            setBBox : function() {
+                var north = parseFloat(this.model.get('north'));
+                var south = parseFloat(this.model.get('south'));
+                var west = parseFloat(this.model.get('west'));
+                var east = parseFloat(this.model.get('east'));
+                this.model.set({mapNorth: north, mapSouth: south, mapEast: east, mapWest: west});
```

Comment:
This should all be taken care of inside the query model. This line: `this.listenTo(this, 'change:north change:south change:east change:west',this.setBBox);` already exists in the query model. Doing this this way in the view is going to break USNG/MGRS.